inst/include/rquantlib_internal.h

```diff
@@ -166,15 +169,17 @@ QuantLib::DateGeneration::Rule getDateGenerationRule(const double n);
 boost::shared_ptr<QuantLib::YieldTermStructure> buildTermStructure(Rcpp::List params, Rcpp::List);
 QuantLib::Schedule getSchedule(Rcpp::List rparam);
 boost::shared_ptr<QuantLib::FixedRateBond> getFixedRateBond(Rcpp::List bondparam, std::vector<double> ratesVec, Rcpp::List scheduleparam);
-boost::shared_ptr<QuantLib::IborIndex> getIborIndex(Rcpp::List index, const QuantLib::Date today);
+//boost::shared_ptr<QuantLib::IborIndex> getIborIndex(Rcpp::List index, const QuantLib::Date today);////******
+/*boost::shared_ptr<QuantLib::IborIndex> getIborIndex(std:string type);////******/
```

Comment:
What are you trying to say here with `//******` ?  To be revisited?

It is a real pain we don't have clean diff's.  I am restoring whitespace to what it was so the diff between the _next_ and the _previous_ diff will be more meaningful.